docs/ceres.md

```diff
@@ -0,0 +1,23 @@
+# nf-core/configs: USDA SCINet Ceres HPC Configuration
+
+> **NB:** You will need an [account](https://scinet.usda.gov/about/signup) to use the Ceres cluster to run the pipeline.
+
+All nf-core pipelines have been successfully configured for use on the Ceres cluster at the United States Department of Agriculture (USDA) Agricultural Research Service (ARS) Scientific Computing Network (SCINet).
+
+To use, run the pipeline with `-profile ceres`. The will download and launch the [`ceres.config`](../conf/ceres.config) which has been pre-configured with a setup suitable for the Ceres cluster. Using this profile will configure Nextflow to download all required software as Singularity images as they are required in the pipeline.
+
+Before running the pipeline, you will need to load Nextflow using the environment module system on Ceres (Singularity is available globally). You can do this by issuing the command:
+
+```bash
+module load nextflow/22.04.3
+```
+
+All of the intermediate files required to run the pipeline will be stored in the `work/` directory by default. It is recommended to use Nextflow's [`-work-dir`](https://nextflow.io/docs/latest/cli.html#run) parameter to change this to a [`/90daydata`](https://scinet.usda.gov/guide/storage/#large-short-term-storage) directory instead, e.g.
+
+```bash
+nextflow run nf-core/mag -profile ceres,test -w /90daydata/shared/$USER/.nextflow/work
+```
+
+Storage in `/90daydata` does not count against your account, but it is deleted 90 days after creation, giving an effective short-term cache for intermediate files in case you wish to re-run a pipeline. All of the main output files will be saved to the directory you specify with `--outdir`.
```

Comment:
This is not specified anywhere in the config itself, maybe put it under a section called `## Recommendations` if you don't want to _force_ a user to put their temporary directories in `/90daydata`

Reply:
This changed a little bit since I initially wrote the config. Updated it to reflect the changes on the cluster.